### ISSUE TITLE
Links on Contributing page need are not a11y compliant

### ISSUE THREAD

**Issue by StephNathai** [yargs/yargs.github.io#78]
**Issue**: Links on the [Yargs Contributing page](https://yargs.js.org/contributing/) are failing the contrast accessibility check due to the fact that _links must be distinguishable without relying on color_ to be a11y compliant.

**Solution**: Underline all 6 links that noncompliant. This will reduce a11y issues on this page by 50%

**Comment by StephNathai** [comment]
@bcoe yep good to close, thanks!
